articles/machine-learning/reference-automl-images-hyperparameters.md

```diff
@@ -104,7 +104,7 @@ The following hyperparameters are for object detection and instance segmentation
 | `box_score_threshold` | During inference, only return proposals with a classification score greater than `box_score_threshold`. <br> Must be a float in the range [0, 1].| 0.3 |
 | `nms_iou_threshold` | IOU (intersection over union) threshold used in non-maximum suppression (NMS) for the prediction head. Used during inference.  <br>Must be a float in the range [0, 1]. | 0.5 |
 | `box_detections_per_image` | Maximum number of detections per image, for all classes. <br> Must be a positive integer.| 100 |
-| `tile_grid_size` | The grid size to use for tiling each image. <br>*Note: tile_grid_size must not be None to enable [small object detection](how-to-use-automl-small-object-detect.md) logic*<br> A tuple of two integers passed as a string. Example: --tile_grid_size "(3, 2)" | No Default |
+| `tile_grid_size` | `tile_grid_size` is not supported for instance segmentation tasks, and will not work with maskrcnn_resnet models. The grid size to use for tiling each image. <br>*Note: tile_grid_size must not be None to enable [small object detection](how-to-use-automl-small-object-detect.md) logic*<br> A tuple of two integers passed as a string. Example: --tile_grid_size "(3, 2)" | No Default |
```

Comment:
Thanks for the PR :)

I'd rearrange this differently: 

The grid size to use for tiling each image. <br>*Note: tile_grid_size must not be None to enable [small object detection](how-to-use-automl-small-object-detect.md) logic. This is not supported for instance segmentation tasks.*<br> A tuple of two integers passed as a string. Example: --tile_grid_size "(3, 2)" | No Default |